multigeiger/transmission.cpp

```diff
@@ -236,15 +232,15 @@ void transmit_data(String tube_type, int tube_nbr, unsigned int dt, unsigned int
     delay(300);
   }
 
-  #if CPU==STICK
-  if(sendToLora && (strcmp(appeui, "") != 0)) {    // send only, if we have LoRa credentials
-    log(INFO, "Sending to TTN ...");
-    displayStatusLine("TTN");
-    send_ttn_geiger(tube_nbr, dt, gm_counts);
-    if (have_thp)
-      send_ttn_thp(temperature, humidity, pressure);
+  if (haveLoRa) {
+    if(sendToLora && (strcmp(appeui, "") != 0)) {    // send only, if we have LoRa credentials
```

Comment:
just add another term to the second "if"?

Reply:
OK